pkg/environment/environment.go

```diff
@@ -59,7 +59,7 @@ func parseEnvs(environ []string) map[string]string {
 
 // MustGetKrewPaths returns ensured index paths for krew.
 func MustGetKrewPaths() KrewPaths {
-	base := filepath.Join(homedir.HomeDir(), ".kube", "plugins", "krew")
+	base := filepath.Join(homedir.HomeDir(), ".krew")
```

Comment:
I don't like krew polluting the homedir. We should respect XDG directories if possible!

Reply:
There's no XDG on macOS though. Not to mention, .kube isn't probably complying with this either?

I think we should take this PR as is, then let people specify a Krew home path through a KREW_ROOT environment variable.